PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -484,6 +516,99 @@ intptr_t CALLBACK WindowsDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM lP
 			break;
 		}
 
+		case WM_CHAR_REPLACEMENT:
+		{
+			if (_hList != NULL)
+			{

```

Comment:
...to remove '{' for reducing the depth of indentation, so the code is more readable.